dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -236,6 +236,7 @@ public synchronized Node getNode() {
         = new LinkedBlockingDeque<SoftReference<Marshaller>>(MAX_LOAD_FACTOR);
     private Deque<SoftReference<Unmarshaller>> unmarshallers 
         = new LinkedBlockingDeque<SoftReference<Unmarshaller>>(MAX_LOAD_FACTOR);
+    private boolean isLoggableFine = LOG.isLoggable(Level.FINE);
```

Comment:
Should this be `final`? `static`?

Reply:
On further thought, I don't think we should cache this check at the class level. If we do, we can't dynamically update the trace without restarting the server/application.

You could cache it at the start of a trace heavy method if performance is concerned.

Reply:
This is already addressed and resolved